eng/pipelines/client.yml

```diff
@@ -20,4 +20,7 @@ resources:
 jobs:
 - template: templates/jobs/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: $(ServiceDirectory)
\ No newline at end of file
+    ServiceDirectory: $(ServiceDirectory)
+    Artifacts: # Prevents Docs steps from Failing on net - client ci
```

Comment:
Will this cause all the docs to generated or none? I think @KarishmaGhiya has made it generate all for the JS client pipeline.

Reply:
None. Except I list all the Dataplane package Details under the Artifacts paramenter